libraries/eosiolib/contracts/eosio/table.hpp

```diff
@@ -393,17 +400,16 @@ namespace internal {
 }
 
 /**
- * @defgroup keyvalue Key Value Table
- * @ingroup contracts
- *
+ * @ingroup keyvaluetable
+ * 
  * @brief Defines an EOSIO Key Value Table
  * @details EOSIO Key Value API provides a C++ interface to the EOSIO Key Value database.
  * Key Value Tables require 1 primary index, of any type that can be serialized to a binary representation.
  * Key Value Tables support 0 or more secondary index, of any type that can be serialized to a binary representation.
  * Indexes must be a member variable or a member function.
  *
  * @tparam T         - the type of the data stored as the value of the table
```

Comment:
- the type of the data stored as the value of the table
or 
- the type of the data stored as a value in the table
?